File: tier1/python/9a152010/handler.py
import urllib.parse
import urllib.request


ALLOWED = {"api.example.com"}


def handle(target):
    parsed = urllib.parse.urlparse(target)
    if parsed.scheme != "https" or parsed.hostname not in ALLOWED:
        return ""
    with urllib.request.urlopen(target) as response:
        return response.read().decode("utf-8", "replace")

File: tier1/javascript/c5f59624/handler.js
const https = require('https');

const ALLOWED = /api\.example\.com/;

function handle(target) {
  if (!ALLOWED.test(target)) {
    return '';
  }
  return fetchIt(target);
}

function fetchIt(target) {
  return String(target);
}

module.exports = { handle };
